onedal/__init__.py

```diff
@@ -42,16 +42,26 @@
 
     _is_dpc_backend = False
 
+_is_spmd_backend = False
+
+if _is_dpc_backend:
+    try:
+        import onedal._onedal_py_spmd_dpc as _spmd_backend
+
+        _is_spmd_backend = True
+    except ImportError:
+        _is_spmd_backend = False
+
```

Comment:
Which is what you do. If `_onedal_py_dpc` fails to import you will exit in line 46. The result is the same as the old `_dpc_backend == False` case. My suggestion saves one `try/except` block, the declaration of one otherwise unused variable, and avoids one level of indentation. I think it is a lot more readable and better to maintain.

Reply:
Now. I fully understand your idea.
So I think my suggestion is better.
There is no need for double importing backends for checking it.
is onedal._onedal_py_dpc doesn't work, no sense to check _onedal_py_spmd_dpc import.
So leaving it as is.